cheri_compressed_cap_common.h

```diff
@@ -67,22 +67,32 @@ _CC_STATIC_ASSERT_SAME(_CC_N(FIELD_INTERNAL_EXPONENT_SIZE) + _CC_N(FIELD_EXP_NON
                            _CC_N(FIELD_EXPONENT_LOW_PART_SIZE),
                        _CC_N(FIELD_EBT_SIZE));
 
+// In order to allow vector loads and store from memory we can optionally reverse the first two fields.
 struct _cc_N(cap) {
     /* offset = cursor - base */
+#ifdef _CC_REVERSE_PESBT_CURSOR_ORDER
+    /* Currently only updated lazily when compress is called.
+     * We might want to make this the canonical field and slice out other fields on demand.*/
+    uint64_t cached_pesbt;
```

Comment:
Still uint64_t...

Reply:
Oh dear, I did a mine / theirs reversal when I rebased. Rebasing + force pushing is a sin IMO.

Reply:
`git diff @{u}` before you force-push...

Reply:
Right, rebase should now now be mismerged